mozilla/blurts-server
# MTNOR-3240

# References:

Jira: MNTOR-3240



# Description
All breaches page not showing breaches in descending order by date.

# How to test
Navigate to /breaches.  Observe that breaches cards are now in descending order by date.

# Checklist (Definition of Done)
